### PR TITLE
Fix NFT double tap to zoom no longer zooms to tapped coordinates

### PR DESCRIPTION
**Pull request by osdnk**
Fixes RNBW-2680

## What changed (plus any additional context for devs)
 I bisected and made random changes 

## PoW (screenshots / screen recordings)

https://streamable.com/fs6rnz

## Dev checklist for QA: what to test

check double tap

### CODE REVIEW

File: src/components/expanded-state/unique-token/ZoomableWrapper.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import Animated, {
   withSpring,
   withTiming,
 } from 'react-native-reanimated';
+import vstyled from 'styled-components';
```
Comment (skylarbarrera) [inline src/components/expanded-state/unique-token/ZoomableWrapper.js:18]: ?

File: src/components/expanded-state/unique-token/ZoomableWrapper.js [hunk 2]
```diff
@@ -43,13 +44,14 @@ const GestureBlocker = styled(View)({
...
 
-const Container = styled(Animated.View)({
-  alignSelf: 'center',
-  shadowColor: ({ theme: { colors } }) => colors.shadowBlack,
-  shadowOffset: { height: 20, width: 0 },
-  shadowOpacity: 0.4,
-  shadowRadius: 30,
-});
+// TODO osdnk
+const Container = vstyled(Animated.View)`
```
Comment (skylarbarrera) [inline src/components/expanded-state/unique-token/ZoomableWrapper.js:48]: I imagine there's a reason for this

**Review by skylarbarrera — APPROVED** [review]
🪄  fine w it

**Review by tchayen — APPROVED** [review]
Works fine. My only concern is that spelling that looks like a typo and weird TODO, but I assume it is intended or you will take care of it.

**Pull request merged**